prime-router/src/test/kotlin/azure/observability/bundleDigest/FhirPathBundleDigestExtractorStrategyTests.kt

```diff
@@ -58,7 +60,12 @@ class FhirPathBundleDigestExtractorStrategyTests {
                             )
                         )
                     ),
-                                listOf("VA"), listOf("MD"), listOf("DC"), "ORU_R01"
+                                listOf("VA"),
+                                listOf(PerformerSummary(performerState = "MD")),
```

Comment:
We should update the test data to include the new fields being populated in the events.